hosts.yml

```diff
@@ -0,0 +1,510 @@
+---
+all:
+  vars:
+    ansible_connection: local
+    ansible_user: ec2-user
+    ansible_become: true
```

Comment:
This file is great, we will eventually want to include it in sample_inventories/ with all the commented lines removed. also use dns names instead of IPs